tools/VscPreviewExtension/src/Proxy/proxy.ts

```diff
@@ -0,0 +1,151 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+import { ExtensionContext, Uri, window, workspace } from "vscode";
+import * as childProcess from "child_process";
+
+import * as ConstVariable from "../constVariables/commonVariables";
+import { Common } from "../common";
+import { DfmService } from "../dfmService";
+import { PreviewType } from "../constVariables/previewType";
+import { ProxyRequest } from "./proxyRequest";
+import { ProxyResponse } from "./proxyResponse";
+import { RequestArray } from "./RequestArray";
+
+export class Proxy {
```

Comment:
requestProxy?

---
In reply to: [113141328](https://github.com/dotnet/docfx/pull/1584#discussion_r113141328) [](ancestors = 113141328)